client/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "client",
+  "version": "0.1.0",
+  "private": true,
+  "engines": {
+    "npm": "6.4.1",
+    "node": "10.13.0"
+  },
+  "dependencies": {
+    "axios": "^0.18.0",
+    "pusher-js": "^4.3.1",
+    "react": "^16.6.3",
+    "react-dom": "^16.6.3",
+    "react-pusher": "^0.2.0",
```

Comment:
This is cool, I didn't even know there was a react library specifically for pusher until you  mentioned in